modules/upgrade/partials/rolling-upgrades/rolling-restart-intro.adoc

```diff
@@ -0,0 +1,68 @@
+ifdef::rolling-upgrade[]
+A rolling upgrade involves putting a broker into xref:manage:node-management.adoc[maintenance mode], upgrading the broker, taking the broker out of maintenance mode, and then repeating the process on the next broker in the cluster. Placing brokers into maintenance mode ensures a smooth upgrade of your cluster while reducing the risk of interruption or degradation in service.
+
+When a broker is placed into maintenance mode, it reassigns its partition leadership to other brokers for all topics that have a replication factor greater than one. Reassigning partition leadership involves _draining_ leadership from the broker and _transferring_ that leadership to another broker.
+
+. Check for topics that have a replication factor greater than one.
++
+If you have topics with `replication.factor=1`, and if you have sufficient disk space, Redpanda Data recommends temporarily increasing the replication factor. This can help limit outages for these topics during the rolling upgrade. Do this before the upgrade to make sure there's time for the data to replicate to other brokers. For more information, see xref:manage:data-migration.adoc#change-topic-replication-factor[Change topic replication factor].
+
+. Ensure that all brokers are active before upgrading:
++
+[,bash]
+----
+rpk redpanda admin brokers list
+----
++
+All brokers should show `active` for `MEMBERSHIP-STATUS` and `true` for `IS-ALIVE`:
++
+.Example output
+[%collapsible]
+====
+[.no-copy]
+```
+NODE-ID  NUM-CORES  MEMBERSHIP-STATUS  IS-ALIVE  BROKER-VERSION
+0        1          active             true      v22.3.11
+1        1          active             true      v22.3.11
+2        1          active             true      v22.3.11
+```
+====
+
+New features in a version are enabled after all brokers in the cluster are upgraded. If problems occur, the upgrade is not committed.
+
+NOTE: Redpanda started supporting xref:develop:consume-data/consumer-offsets.adoc[consumer offsets] in version 22.1. When upgrading from version 21.11 to 22.1, after all brokers are upgraded, Redpanda starts to migrate consumer group topics to `__consumer_offsets`. This takes some time, depending on the data size. Until it finishes, all consumer group-related operations (consume, offset commit, coordinator election) are blocked. The migration to consumer offsets is complete when you see `consumer offset feature enabled` in all brokers.
```

Comment:
```suggestion
```
Suggest deleting this entire note. I don't think we'd support anyone upgrading from 21.11 to 22.1 now.